apps/sensenet/src/components/field-controls/webhook-payload.tsx

```diff
@@ -106,35 +117,15 @@ export const WebhookPayload: React.FC<ReactClientFieldSetting<LongTextFieldSetti
     default:
       return (
         <>
-          <FormLabel style={{ transform: 'translate(0, 1.5px) scale(0.75)', transformOrigin: 'top left' }}>
+          <Typography variant="caption" gutterBottom={true}>
             {localization.webhooksPayload.payload}
-          </FormLabel>
-          <RadioGroup
-            aria-label="payload"
-            name="payload"
-            value={String(useDefault)}
-            onChange={(event) => setUseDefault(event.target.value === 'true')}>
-            <FormControlLabel
-              disabled
-              value="true"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.useDefault}
-            />
-            <FormControlLabel
-              disabled
-              value="false"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.customize}
-            />
-          </RadioGroup>
-          <TextareaAutosize
-            className={classes.textArea}
-            disabled
-            aria-label="minimum height"
-            rowsMin={10}
-            cols={70}
-            value={value}
-          />
+          </Typography>
+          <Typography variant="body1" gutterBottom={true}>
+            {String(useDefault) ? localization.webhooksPayload.defaultPayload : localization.webhooksPayload.customize}
+          </Typography>
+          <Paper elevation={0} className={widgetClasses.container}>
+            {value || ''}
```

Comment:
1. It seems better if you wrap this into a pre tag:
```suggestion
           <pre>{value || ''}</pre>
```
2. We can also render the example if  default payload is selected?! cause nothing is there in this case:
![image](https://user-images.githubusercontent.com/22347393/119157320-5bcbc700-ba55-11eb-8258-25fee1a13a6b.png)

Reply:
I think we cannot display the example as a default value because it is not the default value, if a field has no value it should be empty as in every other case :)